packages/react-storage/src/components/StorageBrowser/adapters/createAmplifyAuthAdapter.ts

```diff
@@ -22,6 +37,11 @@ export const createAmplifyAuthAdapter = (): StorageBrowserAuthAdapter => {
     return Promise.resolve({
       locations: [
         { type: 'BUCKET', permission: 'READWRITE', scope: `s3://${bucket}/*` },
+        {
+          type: 'PREFIX',
+          permission: 'READWRITE',
+          scope: `s3://${bucket}/public/*`,
+        },
```

Comment:
Is this a temporary work around for the `default-auth` path or something like that?